<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OmniComm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="format_table">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Source</string>
       </attribute>
       <widget class="QTabWidget" name="tabWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>191</width>
          <height>172</height>
         </rect>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Serial (COM)</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>COM Port:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Baud:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Timeout:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="serial_timeout_edit">
            <property name="text">
             <string>1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Buffer Size:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="serial_buffsize_edit">
            <property name="text">
             <string>1024</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="serial_connect_btn">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="serial_disconnect_btn">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="serial_baud_edit">
            <property name="text">
             <string>9600</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="serial_port_edit">
            <property name="text">
             <string>9</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>UDP</string>
         </attribute>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Port #:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Bind Addres:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Timeout:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="udp_timeout_edit">
            <property name="text">
             <string>1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Buffer Size:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="udp_buffsize_edit">
            <property name="text">
             <string>1024</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="udp_connect_btn">
            <property name="text">
             <string>Connect</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="udp_disconnect_btn">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="udp_port_edit">
            <property name="text">
             <string>1350</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="udp_bindaddr_edit">
            <property name="text">
             <string>0.0.0.0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>File</string>
         </attribute>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>381</width>
          <height>301</height>
         </rect>
        </property>
        <property name="title">
         <string>Data View</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>View as:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="data_view_encoding_combo">
             <item>
              <property name="text">
               <string>ASCII</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Hex</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="raw_data_textEdit">
           <property name="font">
            <font>
             <family>Consolas</family>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="backgroundVisible">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>TM</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Format String:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="format_lineEdit">
            <property name="text">
             <string>&lt;BlBffffffffff</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>TextLabel:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="labels_lineEdit">
            <property name="text">
             <string>id,seqack,mode,thrust_sfn,hinge_x,hinge_y,rcs0,rcs1,rcs2,rcs3,mass_est,mass_fuel,mass_lox</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Units:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="units_lineEdit">
            <property name="text">
             <string>unknown,unknown,unknown,N,deg,deg,uknown,uknown,uknown,uknown,kg,kg,kg</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="update_format_btn">
          <property name="text">
           <string>Update Packet Format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tm_table">
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="showGrid">
           <bool>false</bool>
          </property>
          <property name="gridStyle">
           <enum>Qt::DashLine</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <property name="rowCount">
           <number>1</number>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>75</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <row/>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Packet_Definition"/>
    <addaction name="actionSave_Packet_Definition"/>
   </widget>
   <widget class="QMenu" name="menuConnect">
    <property name="title">
     <string>Connect</string>
    </property>
    <addaction name="actionSerial_COM"/>
    <addaction name="actionUDP_Socket"/>
    <addaction name="actionFile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConnect"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Packet_Definition">
   <property name="text">
    <string>Load Packet Definition</string>
   </property>
  </action>
  <action name="actionSave_Packet_Definition">
   <property name="text">
    <string>Save Packet Definition</string>
   </property>
  </action>
  <action name="actionSerial_COM">
   <property name="text">
    <string>Serial COM</string>
   </property>
  </action>
  <action name="actionUDP_Socket">
   <property name="text">
    <string>UDP Socket</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="text">
    <string>File</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
